Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -52,18 +52,7 @@ public struct ConvertAction: Action, RecreatingContext {
     
     let sourceRepository: SourceRepository?
     
-    private(set) var context: DocumentationContext {
-        didSet {
```

Comment:
This `didSet` isn't needed because the setter is private and no code path needs to reconfigure the new context.